src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,49 +1,83 @@
+  const [isLoading, setIsLoading] = useState(true);
+  const [errorMessage, setErrorMessage] = useState('');
+
+  useEffect(() => {
+    setErrorMessage('');
+    setIsLoading(true);
+
+    if (selectedTodo) {
+      getUser(selectedTodo?.userId)
+        .then(userFromServer => {
+          setUser(userFromServer);
+          setIsLoading(false);
+        })
+        .catch(error => {
+          setErrorMessage(error.message);
+        })
+        .finally(() => setIsLoading(false));
+    } else {
+      setErrorMessage('No selected todo');
+    }
+  }, [selectedTodo]);
 
-export const TodoModal: React.FC = () => {
   return (
     <div className="modal is-active" data-cy="modal">
       <div className="modal-background" />
 
-      {true ? (
-        <Loader />
-      ) : (
-        <div className="modal-card">
-          <header className="modal-card-head">
-            <div
-              className="modal-card-title has-text-weight-medium"
-              data-cy="modal-header"
-            >
-              Todo #2
-            </div>
+      {isLoading && !user
+        && <Loader />}
+      {!isLoading && !errorMessage
+        && (
+          <div className="modal-card">
+            <header className="modal-card-head">
+              <div
+                className="modal-card-title has-text-weight-medium"
+                data-cy="modal-header"
+              >
+                {`Todo #${selectedTodo?.id}`}
+              </div>
 
-            {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-            <button
-              type="button"
-              className="delete"
-              data-cy="modal-close"
-            />
-          </header>
-
-          <div className="modal-card-body">
-            <p className="block" data-cy="modal-title">
-              quis ut nam facilis et officia qui
-            </p>
-
-            <p className="block" data-cy="modal-user">
-              {/* <strong className="has-text-success">Done</strong> */}
-              <strong className="has-text-danger">Planned</strong>
-
-              {' by '}
-
-              <a href="mailto:Sincere@april.biz">
-                Leanne Graham
-              </a>
-            </p>
+              {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
+              <button
+                type="button"
+                className="delete"
+                onClick={() => onSelectTodo(null)}
+                data-cy="modal-close"
+              />
+            </header>
+
+            <div className="modal-card-body">
+              <p className="block" data-cy="modal-title">
+                {selectedTodo?.title}
+              </p>
+
+              <p className="block" data-cy="modal-user">
+                {selectedTodo?.completed
+                  ? <strong className="has-text-success">Done</strong>
+                  : <strong className="has-text-danger">Planned</strong>}
```

Comment:
let's use single `strong` tag and only change children and className

Reply:
Thanks.